http_logs/index.json

```diff
@@ -11,7 +11,11 @@
     },
     "properties": {
       "@timestamp": {
-        "format": "strict_date_optional_time||epoch_second",
+        {%- if ingest_pipeline is defined and ingest_pipeline == "grok" %}
```

Comment:
shouldn't this conditional also include and `or runtime_fields is defined` as done in https://github.com/elastic/rally-tracks/blob/b1479ad8b466058f6c7348b9a1e875eba7a6447f/http_logs/track.json#L48?

Reply:
The [conditional](https://github.com/elastic/rally-tracks/blob/b1479ad8b466058f6c7348b9a1e875eba7a6447f/http_logs/track.json#L48) you referenced in track.json defines the path for which index mapping to use. The runtime-fields configuration does not use the `index.json` mapping and therefore is not included in the conditional.

Reply:
Makes sense, thanks.